spec/features/incoming_mail_handler_spec.rb

```diff
@@ -0,0 +1,79 @@
+describe "Handles incoming email" do
+  let(:proposal) { FactoryGirl.create(:proposal, :with_parallel_approvers) }
```

Comment:
thoughts on private methods instead of `let` per convo here https://github.com/18F/C2/pull/676#issuecomment-146062260